toucan_connectors/toucan_connector.py

```diff
@@ -548,7 +548,7 @@ def _validate(self, engine_version: str | float | None) -> bool:
         A small validation function for incoming version format
         """
         if engine_version is None:
-            raise UnavailableVersion('This engine has an unavailable version')
+            raise UnavailableVersion  # pragma: no cover
```

Comment:
In case of using `VersionableEngineConnector._validate()` seperately !

Reply:
It's a private method, so this is not supposed to happen